plugins/examples/stable-analysis/src/main/java/org/elasticsearch/example/analysis/lucene/CharTokenizer.java

```diff
@@ -8,12 +8,21 @@
 
 package org.elasticsearch.example.analysis.lucene;
 
-import org.apache.lucene.analysis.util.CharTokenizer;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
 
-public class UnderscoreTokenizer extends CharTokenizer {
+public class CharTokenizer extends org.apache.lucene.analysis.util.CharTokenizer {
```

Comment:
Can we use another name that doesn't cause a need for the fully qualified superclass? Even CustomCharTokenizer would be better than naming the class the same as it's superclass.

Reply:
I will use `CustomCharTokenizer`